components/Blueprints/class-runner.php

```diff
@@ -379,12 +379,7 @@ private function load_blueprint() {
 		// Validate the Blueprint string we've just loaded.
 
 		// **UTF-8 Encoding:** Assert the Blueprint input is UTF-8 encoded.
-		$is_valid_utf8 = false;
-		if ( function_exists( 'mb_check_encoding' ) ) {
-			$is_valid_utf8 = mb_check_encoding( $blueprint_string, 'UTF-8' );
-		} else {
-			$is_valid_utf8 = ! _wp_has_noncharacters_fallback( $blueprint_string );
-		}
+		$is_valid_utf8 = ! wp_has_noncharacters( $blueprint_string );
```

Comment:
I left a note in the other PR making this change, but it’s possible there is a misunderstanding here between _valid UTF-8_ and _noncharacters_, as noncharacters are valid UTF-8, and invalid UTF-8 cannot form noncharacters.

there is, however, `wp_is_valid_utf8()`

Reply:
Ah good point, I mixed it all up. Thank you for catching this.